source-repo-scripts/bump_dependency.bash

```diff
@@ -347,8 +347,10 @@ for ((i = 0; i < "${#LIBRARIES[@]}"; i++)); do
   cd ${TEMP_DIR}/homebrew-simulation
   startFromCleanBranch ${BUMP_BRANCH} master
 
+  # NOTE(CH3): This won't interfere with $LIB=gzXXX so it's fine
   # expand ign-* to ignition-*
   FORMULA_BASE=${LIB/ign/ignition}
```

Comment:
We'll only use this script to bump `gz` now, from Gazebo-H onwards, so I think it's ok to remove all `ign` / `ignition`